starter-code/css/main.css

```diff
@@ -6,3 +6,158 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+.nav {
+    background-color: white;
+    width: 100vw;
+    height: 60px;
+    top: 12px;
+    position: fixed;
+}
+.nav img {
+    margin-top: -4px;
```

Comment:
unless you want something to go off the screen you usually would not add negative margin.